packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -80,7 +87,7 @@ const startTrRoutingProcess = async (
         command,
         commandArgs,
         waitString,
-        useShell: false,
+        useShell: true, // For unknown reason yet, libmemcached pool need trRouting to be run in a shell to work
```

Comment:
Maybe add a FIXME so that we can figure out the reason and document it. What if it's because of a coding error on our part that would need fixing?